package.json

```diff
@@ -4,8 +4,10 @@
   "description": "file-loader that stores a map of original file names to file names with hashes in the compilation stats",
   "main": "index.js",
   "scripts": {
-    "test": "env NODE_ENV=test mocha --require ./test/power_assert_loader.js test/basic.js test/integration.js --watch",
-    "test-ci": "env NODE_ENV=test mocha --require ./test/power_assert_loader.js test/basic.js test/integration.js"
+    "lint": "standard",
+    "lint-fix": "standard --fix",
+    "test": "npm run lint && env NODE_ENV=test mocha --require ./test/power_assert_loader.js test/basic.js test/integration.js --watch",
```

Comment:
Hm, maybe it's better to create separate `lint-watch` task? There is an amazing https://www.npmjs.com/package/eslint-watch that does this job well.

As a side note. I think that the previous task naming is unfortunate. If I (or let's say CI) run `npm test` I expect to run it in a single mode. Maybe we should move watch mode to `npm run test-watch` for consistency sake?